### ISSUE TITLE
MEGAsync tray icon not showing on Raspbian Jessie

### ISSUE THREAD

**Issue by JoaquimParis** [meganz/MEGAsync#225]
Hi There.
Linux "senior Rookie" here.
The MEGAsync icon stopped showing on my system (PI3, Raspbian Jessie).
I've tried the _lxpanelctl restart_ but didn't work.
I've tried editing the _megasync.desktop_ (_Exec=bash -c "sleep 20 ; megasync"_) file but didn't work either.
Please help.

In addition, how do I check if the application is running?

**Comment by polmr** [comment]
Unable to reproduce in an up-to-date Rasbian 8. Please ensure you have your RPI updated (`apt-get update && apt-get upgrade`), and are using latest MEGAsync. You can check megasync is running typing in a console:
```
pgrep megasync
```
You can also execute megasync again (`megasync` in a console, or Menu->Internet->MEGAsync). This should open the status UI, so you can interact with MEGAsync even in the absence of tray icon.

**Comment by JoaquimParis** [comment]
hi polmr.

Thanks for the reply.
I've updated and upgraded the system (including megasync). No good.

Killing the process and restarting it manually everytime solved the problem but that's not the solution I'm looking for (just as aditional info).

any clues?

**Comment by polmr** [comment]
So ...  upon startup megasync won't load the tray icon. If you kill it and start it manually it does load it. But if you set a timeout at `~/.config/autostart/megasync.desktop` via `Exec=bash -c "sleep 20 ; megasync"`, it does not load the icon either. I can only assume 20 seconds was not enough. You might want to try a longer timeout. Also, ensure you are editing the `~/.config/autostart/megasync.desktop`, not `/usr/share/applications/megasync.desktop`

**Comment by JoaquimParis** [comment]
The ~/.config wasn't the one I was editing. Good to learn something new. :+1: 

New info I forgot to mention: I'm accessing the Pi through TightVNC.
Does that count somehow or it makes no diff?

**Comment by polmr** [comment]
No, I don't think so. Did the delay in `~/.config/autostart/megasync.desktop` work?

**Comment by cicciopizza** [comment]
> No, I don't think so. Did the delay in `~/.config/autostart/megasync.desktop` work?

This work for me :ok_hand:
